<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gpio_Ctrl</class>
 <widget class="QDialog" name="Gpio_Ctrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>92</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>92</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GPIO控制</string>
  </property>
  <widget class="QCheckBox" name="checkBox_GPIO1">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>16</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GPIO1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_GPIO3">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>16</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GPIO3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_GPIO4">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>16</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GPIO4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_GPIO2">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>16</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GPIO2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_GPIO5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>16</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GPIO5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
